### ISSUE TITLE
IINA full window is crashed on MacOS Big Sure.

### ISSUE THREAD

**Issue by EricGolovin** [iina/iina#3027]

**Comment by lhc70000** [comment]
So does it crash when entering full screen?

**Comment by saagarjha** [comment]
I can't seem to reproduce this. Would you mind providing a couple more details, and a crash log if you have it?

**Comment by EricGolovin** [comment]
No, it does not crash. UI becomes junky. There is no option to enter full screen mode; only when I double tap on the player window, it somehow enters full screen mode with none hidden title and control buttons on the upper left corner.

![somehow-fullScreen](https://user-images.githubusercontent.com/19877687/86359503-c0cdd100-bc79-11ea-8e12-e49b74692c59.jpg)

**Comment by lhc70000** [comment]
Are you using legacy full screen?

> There is no option to enter full screen mode

Was there an option before? You can enter full screen using the menu, keyboard shortcuts and the toolbar button (if you added it to the on screen controller), do you mean that they all stopped working?

**Comment by malhal** [comment]
Open movie, double click to full screen, double click to go back to window. Double click to full screen is now disabled.
Also trackpad gesture left/right to scrub through movie is now disabled.

1.1.1 Build 125
BigSur 11.0.1 (20B29)

**Comment by markkuossi** [comment]
@lhc70000 
@saagarjha 

Indeed, this happens to me too in Big Sur. In Catalina I did not have this issue.

It occurs exactly like @malhal described above.

And only with legacy full screen selected. Unselect legacy, and the issue goes away.

But for IINA I prefer legacy full screen mode.

Would be nice if this was fixed.
